File: Basic Problem Solving/isPrime/isPrime.js
const isPrime = (num) => {
    if (num == 2) return "2 is niether be prime nor be composite"
    let sqrt = Math.floor(Math.sqrt(num))
    for(let i = 2 ; i <= sqrt; i++){
        if(num%i==0){
            return false
        }
    }
    return true
}

console.log(isPrime(36))
console.log(isPrime(17))
console.log(isPrime(3))
console.log(isPrime(1483))

File: Basic Problem Solving/isPrime/isPrime.py
import math

def isPrime(num):
    if (num == 2):
        return "2 is niether be prime nor be composite"
    sqrt = int(math.sqrt(num))
    for i in range(2,sqrt + 1):
        if num%i == 0:
            return False
    return True

print(isPrime(36))
print(isPrime(17))
print(isPrime(3))

